Add modules

tingamapuro04 opened the pull request:
I have worked on the classes-Developments branch where I have introduced modules directory where i have added two two modules. The two modules have been exported to the index.js file. In the luxon.js i have added script downloaded online to it and I have added it as a module in the index.js file to display the current date and time below the navbar.

tingamapuro04 committed:
Add modules

tingamapuro04 committed:
Remove unused code

omar-muhamad commented on index.js:
```diff
@@ -0,0 +1,67 @@
+  awesomeH1.style.display = 'none';
+  form.style.display = 'none';
+});
+
+date.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
```
- [ ] [Optional] [Highly Recommended] Please add `window.onload` function to get the books from the local storage and display them because now if you try to refresh the page after adding some books everything disappears from the page.

omar-muhamad reviewed (changes requested):
# Changes Required ♻️
### Hi @tingamapuro04,

Nice job, you are almost there.

![](https://camo.githubusercontent.com/56bcd754643682fcc078b66eaa7278dead6c9c5013595c875b4ddcbe0f0ef0aa/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f78554f786668365a4d373565664d3342716f2f67697068792d646f776e73697a65642d6d656469756d2e676966)

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please fix the `removeBook` method because when I click to delete a certain book it deletes another book. You can use `array.filter` instead I think it will fix the problem.

https://github.com/tingamapuro04/awesome-Books/blob/b594106092c5e9bd5de3d8f260c6e778faac6b57/modules/index1.js#L63

## [Optional] suggestions:

-[ ] Please check the inline comments below.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

tingamapuro04 committed:
window.onload

tingamapuro04 committed:
Fix remove button

tingamapuro04 committed:
Delete from local storage

tingamapuro04 committed:
Fix linter errors

tingamapuro04 committed:
Linter errors

tingamapuro04 commented:
As suggested I have modified my function such that I can now remove the targeted book from the local storage. I have also added some styling to my project

phelian23 reviewed (approved):
Hi @tingamapuro04,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
